website/content/docs/configuration/client.mdx

```diff
@@ -147,12 +147,17 @@ client {
   configuration.
 
 - `bridge_network_name` `(string: "nomad")` - Sets the name of the bridge to be
-  created by nomad for allocations running with bridge networking mode on the
+  created by Nomad for allocations running with bridge networking mode on the
   client.
 
 - `bridge_network_subnet` `(string: "172.26.64.0/20")` - Specifies the subnet
   which the client will use to allocate IP addresses from.
 
+- `bridge_network_hairpin_mode` `(bool: false)` - Specifies if hairpin mode
+  is enabled on the network bridge created by Nomad for allocations running
+  with bridge networking mode on this client. You may use the corresponding
+  node attribute `nomad.bridge.hairpin_mode` in constraints.
```

Comment:
```suggestion
- `bridge_network_hairpin_mode` `(bool: false)` - Specifies if hairpin mode
  is enabled on the network bridge created by Nomad for allocations running
  with bridge networking mode on this client. You may use the corresponding
  node attribute `nomad.bridge.hairpin_mode` in constraints. When hairpin mode 
  is enabled allocations are able to reach their own IP and port. Changing this
  value requires a reboot of the client host to take effect.
```

I think? 😅 

Feel free to modify any of this.

Reply:
For now, I think I'd like to omit the additional text so we can get the merge done. Then test the behavior around this on a running system, and then if needed come back and add it in a followup PR. I've added #16023 to track this